SignalMessaging/environment/OWSSounds.m

```diff
@@ -0,0 +1,463 @@
+            return @"aurora.m4r";
+        case OWSSound_Bamboo:
+            return @"bamboo.m4r";
+        case OWSSound_Chord:
+            return @"chord.m4r";
+        case OWSSound_Circles:
+            return @"circles.m4r";
+        case OWSSound_Complete:
+            return @"complete.m4r";
+        case OWSSound_Hello:
+            return @"hello.m4r";
+        case OWSSound_Input:
+            return @"input.m4r";
+        case OWSSound_Keys:
+            return @"keys.m4r";
+        case OWSSound_Note:
+            return @"note.m4r";
+        case OWSSound_Popcorn:
+            return @"popcorn.m4r";
+        case OWSSound_Pulse:
+            return @"pulse.m4r";
+        case OWSSound_Synth:
+            return @"synth.m4r";
+
+            // Ringtone Sounds
+        case OWSSound_Apex:
+            return @"Apex.m4r";
+        case OWSSound_Beacon:
+            return @"Beacon.m4r";
+        case OWSSound_Bulletin:
+            return @"Bulletin.m4r";
+        case OWSSound_By_The_Seaside:
+            return @"By The Seaside.m4r";
+        case OWSSound_Chimes:
+            return @"Chimes.m4r";
+        case OWSSound_Circuit:
+            return @"Circuit.m4r";
+        case OWSSound_Constellation:
+            return @"Constellation.m4r";
+        case OWSSound_Cosmic:
+            return @"Cosmic.m4r";
+        case OWSSound_Crystals:
+            return @"Crystals.m4r";
+        case OWSSound_Hillside:
+            return @"Hillside.m4r";
+        case OWSSound_Illuminate:
+            return @"Illuminate.m4r";
+        case OWSSound_Night_Owl:
+            return @"Night Owl.m4r";
+        case OWSSound_Opening:
+            return @"Opening.m4r";
+        case OWSSound_Playtime:
+            return @"Playtime.m4r";
+        case OWSSound_Presto:
+            return @"Presto.m4r";
+        case OWSSound_Radar:
+            return @"Radar.m4r";
+        case OWSSound_Radiate:
+            return @"Radiate.m4r";
+        case OWSSound_Ripples:
+            return @"Ripples.m4r";
+        case OWSSound_Sencha:
+            return @"Sencha.m4r";
+        case OWSSound_Signal:
+            return @"Signal.m4r";
+        case OWSSound_Silk:
+            return @"Silk.m4r";
+        case OWSSound_Slow_Rise:
+            return @"Slow Rise.m4r";
+        case OWSSound_Stargaze:
+            return @"Stargaze.m4r";
+        case OWSSound_Summit:
+            return @"Summit.m4r";
+        case OWSSound_Twinkle:
+            return @"Twinkle.m4r";
+        case OWSSound_Uplift:
+            return @"Uplift.m4r";
+        case OWSSound_Waves:
+            return @"Waves.m4r";
+
+            // Calls
+        case OWSSound_CallConnecting:
+            return @"ringback_tone_cept.caf";
+        case OWSSound_CallOutboundRinging:
+            return @"ringback_tone_ansi.caf";
+        case OWSSound_CallBusy:
+            return @"busy_tone_ansi.caf";
+        case OWSSound_CallFailure:
+            return @"end_call_tone_cept.caf";
```

Comment:
Some of these replacement sounds for the "call sounds" were more obvious than others.  To be discussed.